spec/requests/api/v2/people_controller_index_spec.rb

```diff
@@ -0,0 +1,213 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::PeopleController do
+  let(:supplier) { create(:supplier) }
+  let!(:application) { create(:application, owner_id: supplier.id) }
+  let!(:access_token) { create(:access_token, application: application).token }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+  let(:response_json) { JSON.parse(response.body) }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+
+  describe 'GET /people' do
+    let(:schema) { load_yaml_schema('get_people_responses.yaml', version: 'v2') }
+    let!(:people) { create_list :person, 2 }
+    let(:params) { {} }
+
+    context 'when successful' do
+      context 'when there are no params' do
+        before { get '/api/v2/people', params: params, headers: headers }
+
+        it_behaves_like 'an endpoint that responds with success 200'
+
+        it 'returns correct attributes' do
+          expect(response_json['data'].first['attributes']).to include(
+            'first_names',
+            'last_name',
+            'date_of_birth',
+            'gender_additional_information',
+            'nomis_prison_number',
+            'prison_number',
+            'criminal_records_office',
+            'police_national_computer',
+          )
+        end
+
+        it 'returns the correct number of people' do
+          expect(response_json['data'].count).to eq(2)
+        end
+      end
+
+      describe 'filtering results by police_national_computer' do
+        let!(:person) { create(:person, police_national_computer: 'AB/1234567') }
+        let(:filters) do
+          {
+            bar: 'bar',
+            police_national_computer: 'AB/1234567',
+            foo: 'foo',
+          }
+        end
+        let(:params) { { filter: filters } }
+
+        before { get '/api/v2/people', params: params, headers: headers }
+
+        it 'returns the correct number of people' do
```

Comment:
Given we truncate the db between examples I think it makes sense for either of the setup blocks (`let!` and `before`) to create another person with a different pnc number.

Reply:
Hey @willfish, we do want to test that is the same person, it's like there was a single test like this:
```
        it 'returns the person' do
          expect(response_json['data'].size).to eq(1)
          expect(response_json).to include_json(data: [{ id: person.id }])
        end
```